driver/src/main/java/org/neo4j/driver/v1/Value.java

```diff
@@ -212,6 +237,12 @@
      */
     String asString();
 
+    /**
+     * @param defaultValue return this value if the value is null.
```

Comment:
Maybe add description and `@throws Uncoercible if value types are incompatible.`

Reply:
Done